pyproject.toml

```diff
@@ -1,3 +1,30 @@
+# pyproject.toml
+[build-system]
+requires      = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python_open_feature_sdk"
+version = "0.0.1"
+description = "Standardizing Feature Flagging for Everyone"
+readme = "README.md"
+authors = [{ name = "Real Python", email = "info@realpython.com" }]
```

Comment:
This will need updating - looks like this was probably copy / pasted from somewhere?